doc/api.rst

```diff
@@ -14,8 +14,6 @@ Classes
 .. autosummary::
    :toctree: generated/
 
-   GlobalAutoReject
-   LocalAutoReject
```

Comment:
make LocalAutoReject private?

Reply:
yes! If people want they can still pass a list of single element to `LocalAutoRejectCV` ...